website/docs/installation/upgrade.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Upgrade authentik
+---
+
+Upgrading to the latest version of authentik, whether a new major release or a patch, involves running a few commands to pull down the latest packages and then restart the servers and databases.
+
+:::warning
+The version of the authentik instance and any outposts must be the same. We recommended that you always upgrade any outposts at the same time you upgrade your authentik instance.
+:::
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<Tabs
+defaultValue="docker-compose"
+values={[
+{label: 'Docker Compose', value: 'docker-compose'},
+{label: 'Kubernetes', value: 'kubernetes'},
+]}
+groupId="platform">
+<TabItem value="docker-compose">
+In your terminal, navigate to your installation directory and run the following commands:
+
+```shell
+docker compose pull
+docker compose up -d
+```
+
+  </TabItem>
+
+  <TabItem value="kubernetes">
+In your terminal, navigate to your installation directory and run the following commands:
+
+```
+helm repo update
+helm upgrade authentik authentik/authentik -f values.yaml
+```
+
+  </TabItem>
+</Tabs>
+
+## Verify your upgrade
+
+You can view the current version of your authentik instance by logging in to the Admin interface, and then navigating to **Dashboards -> Overview**.
+
+![](./version1.png)
+
+## Tips and troubleshooting your upgrade
+
+<Tabs
+defaultValue="docker-compose"
+values={[
+{label: 'Docker Compose', value: 'docker-compose'},
+{label: 'Kubernetes', value: 'kubernetes'},
+]}
+groupId="platform">
+<TabItem value="docker-compose">
+If you run the upgrade commands but your version on the Dashboard doesn’t change,you might need to make sure the old `authinstall database`` is removed before upgrading.
```

Comment:
Yeah... I had this as a Note for myself, because I ran into this problem and running `docker compose down -v` did resolve it... but I am not confident of which database it removed. 

Are you proposing that we shold recommend running `docker compose up --build -d `? As a troubleshooting tip, or...?